Change feature links to networks

zabio3 opened the pull request:
Using `--link` flag is a legacy feature of docker.

https://docs.docker.com/network/links/

zabio3 committed:
Change components links to netwokrs

uLan08 commented:
There's actually no need to create an external network as `docker-compose` creates one by default and all of the services defined in the compose file are connected to it automatically.

Commit:
Modify readme

There's actually no need to create an external network as docker-compose creates one by default and all of the services defined in the compose file are connected to it automatically. by uLan08

DanielDent commented:
Thanks for opening this - I'll re-visit this issue after the -dev branch lands on master.
